guide-us-english.tex

```diff
@@ -166,7 +166,7 @@ \subsubsection{Scrum of Scrums Master (SoSM)}
 	\item make an impediment backlog visible to the organization.
 	\item remove impediments that the teams cannot address themselves.
 	\item facilitate prioritization of impediments, with particular attention to cross-team dependencies and the distribution of backlog.
-	\item improve the efficacy of the Scrum of Scrums.
+	\item improve the effectiveness of the Scrum of Scrums.
```

Comment:
Yeah, makes sense. Looked it up. They're synonyms, and effectiveness is the better known word.